helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -143,12 +142,12 @@ internal class ListLicensesCommand : CliktCommand(
                 ortResult.getExcludes().paths
             } else {
                 packageConfigurationProvider.getPackageConfiguration(packageId, provenance)?.pathExcludes.orEmpty()
-            }.any { it.matches(path)}
+            }.any { it.matches(path) }
```

Comment:
This should be fixed in the "Handle path excludes for packages" commit that introduced it.